modules/serverless-build-func-kn.adoc

```diff
@@ -39,7 +62,21 @@ Building function image
 Function image has been built, image: quay.io/username/example-function:latest
 ----
 
-To learn more about `kn func build` command options, you can use the help command:
+[id="serverless-build-func-kn-push_{context}"]
+== Push flag
+
+You can add the `--push` flag to a `kn func build` command to attempt to automatically push the function image after it is successfully built:
```

Comment:
I think it might be good to remove "to attempt" just for clarity. Unless it is something that doesn't work every time, then I might add another sentence that states that the push could fail and why it would. 
```suggestion
You can add the `--push` flag to a `kn func build` command to automatically push the function image after it is successfully built:
```

Reply:
I'd say there are a number of reasons why, and we wouldn't be able to explain all of them, since they would be individual build errors for code that someone wrote as a function in any number of languages. I'd prefer to leave it as is if that's OK - I believe that this would make sense for the intended audience.